Restructure documentation (#193)
xKDR/Survey.jl: docs/src/getting_started.md

```@repl tutorial
apisrs = load_data("apisrs")
```

`apisrs` is a simple random sample of the Academic Performance Index of Californian
schools. The [`load_data`](@ref) function loads it as a
[`DataFrame`](https://dataframes.juliadata.org/stable/lib/types/#DataFrames.DataFrame).
You can look at the column names of `apisrs` to get an idea of what the dataset
contains.

```@repl tutorial
names(apisrs)
```

Next, build a survey design from your `DataFrame`:

```@repl tutorial
srs = SurveyDesign(apisrs; weights=:pw)
```

This is a simple random sample design with weights given by the column `:pw` of
`apisrs`. You can also create more complex designs such as stratified or cluster
sample designs. You can find more information on the complete capabilities of
the package in the [Manual](@ref manual). The purpose of this tutorial is to show the
basic usage of the package. For that, we will stick with a simple random sample.

Now you can analyse your design according to your needs using the
[functionality](@ref Index) provided by the package. For example, you can compute
the estimated mean or population total for a given variable. Let's say we're

> interested in the mean Academic Performance Index from the year 1999. First we
> need to convert the [`SurveyDesign`](@ref) to a [`ReplicateDesign`](@ref) using
> bootstrapping:

Reviewer: ```suggestion
interested in the mean Academic Performance Index from the year 1999. First we
convert the [`SurveyDesign`](@ref) to a [`ReplicateDesign`](@ref) using
bootstrapping, which allows us to calculate the standard error of estimators:
```

Author: Rephrased, see the new version.

```@repl tutorial

> bsrs = bootweights(srs)

Reviewer: ```suggestion
bsrs = bootweights(srs; replicates = 1000)
```

```

> We do this because [TODO: explain why]. Now we can compute the estimated mean:

Reviewer: ```suggestion
Now, we can estimate the mean with it's standard error:
```

Reviewer: @smishr is this correct language?

Author: I rephrased this, see the new version.
